pkgs/development/web/bun/default.nix

```diff
@@ -4,31 +4,31 @@ let
     aarch64-darwin = {
       arch = "aarch64";
       shortName = "darwin";
-      sha256 = "c82547d96125bf93ae76dafe203cae5f7cd50d041bfb1cf972f9f0232a0d1cc1";
+      sha256 = "1pi9k5bipcgay8r2vppxclb5hiy7cf0fiwsn5xgjqgdsh1mwcpj7";
     };
 
     aarch64-linux = {
       arch = "aarch64";
       shortName = "linux";
-      sha256 = "3430f3ff456ee86ddb607a46ee937c9c1a02b8e4d2546de52b4493878f66afb8";
+      sha256 = "14x31bbnm2yra0fxyy4c4r7y0q605m3q3c2dfkpd8w2jyqz8anh9";
```

Comment:
```suggestion
      sha256 = "sha256-KSC4gdsJZJoPjMEs+VigVuqlUDhg4sL054WRlAbB+eA=";
```

Reply:
I see, so instead of `nix-prefetch-url $URL` one has to use:
`nix store prefetch-file --json $URL | jq '.hash'
I'll patch the other hashes as well. Thanks!

Reply:
You can also use `lib.fakeHash`/`lib.fakeSha256` and it'll tell you what the expected hash was. It only silently fails if it has the store path belonging to the hash you gave it in-store.

Reply:
Thanks, I was aware of that, but I was unsure how to trigger builds for other architecture's other than my own.

Given the lazy evaluation of Nix I thought it wouldn't fail without that.

Reply:
`nix build .#legacyPackages.aarch64-linux.hello` if your machine has `boot.binfmt.emulatedSystems` set. I think `--option system aarch64-linux` with the old CLI but not sure about that one.

Reply:
Thanks, that worked once configured! Sadly there is no darwin qemu emulation yet, so have to stick with the `nix store` one for those sources.